app/grandchallenge/workstation_configs/templates/workstation_configs/workstationconfig_detail.html

```diff
@@ -279,6 +292,18 @@ <h2>Linking Configuration</h2>
     {% endif %}
 {% endblock %}
 
+{% block extra_css %}
+    {{  block.super }}
+    <style>
+        .color-box {
+            display: inline-block;
+            border: black solid 1px;
+            min-width: 0.9375rem;
+            min-height: 0.9375rem;
+        }
```

Comment:
Move this to our css files, the block is only there for includes from other applications, not for inlines.

Reply:
Done.